OSDOCS-9959: NetObserv Health dashboard updates (#75319)
openshift/openshift-docs: modules/network-observability-health-alerts-overview.adoc

// Module included in the following assemblies:
//
// * network_observability/network-observability-operator-monitoring.adoc

:_mod-docs-content-type: CONCEPT
[id="network-observability-health-alert-overview_{context}"]
= Health alerts

> A health alert banner that directs you to the dashboard can appear on the *Network Traffic* and *Home* pages in the event that an alert is triggered. Alerts are generated in the following cases:

Reviewer: ```suggestion
A health alert banner that directs you to the dashboard can appear on the *Network Traffic* and *Home* pages if an alert is triggered. Alerts are generated in the following cases:
```

* The `NetObservLokiError` alert occurs if the `flowlogs-pipeline` workload is dropping flows because of Loki errors, such as if the Loki ingestion rate limit has been reached.
* The `NetObservNoFlows` alert occurs if no flows are ingested for a certain amount of time.

> * The `NetObservAgentFlowsDropped` alert occurs if the Network Observability eBPF agent hashmap table is full, and the eBPF agent processes flows at a degraded performance.

Reviewer 2: we can get that alert for ebpf limiter which is golang infra condition i.e nothing can be done I would watch https://github.com/netobserv/network-observability-operator/pull/640 to see where it will end in case we need to mention both conditions

cc'd @jotak

Author: @msherif1234 @jotak ok I added the piece to the `NetObservAgentFlowsDropped` about the capacity limit being triggered. one question that came up in SCRUM is should this alert be named `NetObservDroppedFlows`? 

Just talking with @msherif1234 in Slack DM now, and he said he agrees the names should be consistent in the console and in FC, so just let me know which one you choose to go with. Makes no diff to me as long as they're the same.

Author: Ok in Slack Mohamed tells me he's going to go with the `Agent` naming :)

Reviewer 3: yeah I opened a bug about this name mismatch: https://issues.redhat.com/browse/NETOBSERV-1653

> * The `NetObservFlowsDropped` alert occurs if the Network Observability eBPF agent hashmap table is full, and the eBPF agent processes flows at a degraded performance, or when the capacity limiter is triggered.

Reviewer: ```suggestion
* The `NetObservFlowsDropped` alert occurs if the Network Observability eBPF agent hashmap table is full, and the eBPF agent processes flows with degraded performance, or when the capacity limiter is triggered.
```
